### DISCUSSION TITLE
Can we have a temporary alternative version of youtube-dl's formula, since its GitHub repo got DMCAed?

### DISCUSSION THREAD

**Discussion by 3719e04** [Homebrew/discussions#90] [Everyday usage]
As mentioned in https://github.com/github/dmca/blob/master/2020/10/2020-10-23-RIAA.md, the GitHub repo of `youtube-dl` at https://github.com/ytdl-org/youtube-dl is taken down. It may affect the formula https://github.com/Homebrew/homebrew-core/blob/master/Formula/youtube-dl.rb and other formulas that depend on `youtube-dl`, for example, `mpv`.

Do we have a protocols to handle this kind of situation?

**Comment by SMillerDev** [comment]
I'm not sure how we could have an alternative for something that doesn't exist anymore. Best option is to disable the formula so people get an explanation instead of just an error

**Comment by Rylan12** [comment]
See https://github.com/Homebrew/homebrew-core/pull/63415
